config/endpoints/13_dgfip_liasses_fiscales.yml

```diff
@@ -30,19 +28,52 @@
     - 'dgfip'
   keywords:
     - 'finances'
+    - 'compte'
     - 'financier'
     - 'résultat'
     - 'exercice'
+    - 'actif'
+    - 'passif'
+    - 'capital'
+    - 'valeur ajoutée'
+    - 'crédit'
+    - 'immobilisation'
+    - 'provision'
+    - 'participation'
+    - 'amortissement'
+    - 'créance'
+    - 'dette'
+    - 'déficit'
+    - 'plus-value'
+    - 'frais'
+    - 'adresse'
     - 'imposition'
     - 'régime'
     - 'déclaration'
     - 'fraude'
+    - 'imprime'
+    - 'bénéficiaire'
+    - 'effectif'
+    - 'déclarant'
+    - 'filiale'
+    - 'société-mère'
+    - 'participation'
   data:
     description: |+
-      Cette API délivre les liasses fiscales d'une entreprise pour l'année renseignée. Pour chaque liasse fiscale, les informations suivantes sont transmises :
-      - Les règles d'imposition applicables à l'entreprise ;
-      - Des informations sur le formulaire complêté par l'entreprise, à l'origine des données transmises ;
-      - Les données de la déclaration.
+      Cette API délivre les liasses fiscales d'une entreprise pour l'année renseignée. Pour chaque numéro d'imprimé, les règles d'imposition applicables à l'entreprise, ainsi que le millésime, la durée de l'exercice en jours et sa date de fin, ou encore la date de dépôt de la déclaration, sont transmis.
+    
+      **Les données présentes dans les imprimés sont variables et de différente nature**&nbsp;: 
+        - des **informations financières, fiscales et comptables** ;
+        - des **données sur l'entreprise et son réseau** (filiale, société-mère, participation) dont les effectifs. 
```

Comment:
on a aussi les effectifs du coup ?!

Reply:
Oui quelques-uns ... faudrait creuser en détail pour qui  et dans quel cadre

Reply:
Je peux creuser plus si tu veux Alexis. Mais en l’état est ce que ça te convient ? Si oui il me faudra ta validation pour pouvoir merger autrement je crois que c’est bloqué

Reply:
imo go merge, on itérera